pkg/visitors/repack.go

```diff
@@ -116,6 +116,9 @@ func createVolumeImageFile(cs *uefi.Section) (*uefi.File, error) {
 	f := &uefi.File{}
 
 	f.Header.Type = uefi.FVFileTypeVolumeImage
+	// set state to valid
+	f.Header.State = 0x07 ^ uefi.Attributes.ErasePolarity
```

Comment:
is the 7 defined anywhere or this another mysterious UEFI undocumented magic number?

Reply:
I guess
https://github.com/tianocore/edk2/blob/e63d54db952634c2a3a6a24b6c2a4c6ff66f22bd/MdePkg/Include/Pi/PiFirmwareFile.h#L94

If we want to fix that 7 it need to be fixed at every location in utk

Reply:
well 3 location (one being commented out)